JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -148,5 +148,14 @@
                 && columnNumber == 0
                 && error == null;
         }
+
+        /**
+        * Returns string representation of an object suitable for diagnostics logging.
+        */
+        public static dump(object: any): string {
+            var objectTypeDump: string = Object.prototype.toString.call(object);
+            var propertyValueDump: string = JSON.stringify(object);
```

Comment:
Since this code is only invoked from the window.onerror the cases where an arbitrary large/"unsafe" object can come to this function is limited to:
- no CORS \ arbitrary object is thrown from the script
- arbitary object is thrown from inline script on the page

This outside of scope of this bug. 

Since the result of this fix will be consumed by us as part of weekly telemetry review, the "big arbitrary object" issue will come up and be noticed - and we'll address it if it meets the bar.